lib/web_ui/lib/src/engine/browser_detection.dart

```diff
@@ -268,6 +268,4 @@ int _detectWebGLVersion() {
 }
 
 /// Whether the current browser supports the Chromium variant of CanvasKit.
-const bool browserSupportsCanvaskitChromium = false;
-// TODO(mdebbar): Uncomment this to enable real detection of browser support.
-// final bool browserSupportsCanvaskitChromium = domIntl.v8BreakIterator != null;
+final bool browserSupportsCanvaskitChromium = domIntl.v8BreakIterator != null;
```

Comment:
If `canvaskit_chromium` also removes the wasm image codecs, then this should also add `&& browserSupportsImageDecoder`. Although in practice, I'm not aware of any Chromium-based browser not supporting image decoders.

Reply:
Agreed.

Reply:
https://github.com/flutter/flutter/issues/122331